frontend/src/pages/NewChainLocation.js

```diff
@@ -13,25 +15,32 @@ import Typography from "@material-ui/core/Typography";
 import Button from "@material-ui/core/Button";
 import Card from "@material-ui/core/Card";
 import CardActions from "@material-ui/core/CardActions";
-import CardContent from "@material-ui/core/CardContent";
-import TextField from "@material-ui/core/TextField";
-import FormControl from "@material-ui/core/FormControl";
-import FormGroup from "@material-ui/core/FormGroup";
-import FormControlLabel from "@material-ui/core/FormControlLabel";
 import Checkbox from "@material-ui/core/Checkbox";
 import { Helmet } from "react-helmet";
+import { CheckboxField, TextForm } from "../components/FormFields";
+import { Alert } from "@material-ui/lab";
+import { CardContent } from "@material-ui/core";
 
 // eslint-disable-next-line import/no-webpack-loader-syntax
-mapboxgl.workerClass = require('worker-loader!mapbox-gl/dist/mapbox-gl-csp-worker').default;
+mapboxgl.workerClass = require("worker-loader!mapbox-gl/dist/mapbox-gl-csp-worker").default;
 
 const accessToken = {
   mapboxApiAccessToken: process.env.REACT_APP_MAPBOX_KEY,
 };
 
-const categories = [
+const categoriesList = [
   { gender: "women" },
   { gender: "men" },
-  { gender: "no gender" },
+  { gender: "kid" },
+];
+
+const sizesList = [
+  { size: "xs" },
+  { size: "s" },
+  { size: "m" },
+  { size: "l" },
+  { size: "xl" },
+  { size: "xxl" },
```

Comment:
In the other form all the sizes were uppercase. You might want to put these category lists in some definition somewhere else, so the forms can share them. Getting them wrong can be a bit of an issue for the data in the database.